pkg/availability/multisigcollector/parts/certcreation/certcreation.go

```diff
@@ -134,8 +134,8 @@ func IncludeCreatingCertificates(
 			return nil
 		}
 
-		if !certificate.receivedSig[from] {
-			certificate.receivedSig[from] = true
+		if !certificate.ReceivedSig[from] {
+			certificate.ReceivedSig[from] = true
```

Comment:
Are these public struct fields being used anywhere outside the package?

Reply:
No, but we access them directly as `certificate.ReceivedSig`, not through a method of `certificate`. In fact, I don't know what the official convention is (should look that up), but I tend to expose fields accessed like this. But I'm ready to be corrected.

Reply:
My understanding is that we should just keep them private if not used outside its package. But not a massive thing either way.

Reply:
Fair enough. To be consistent, I un-exported everything that is not used outside of the package.